app/views/ioj_passported.html

```diff
@@ -10,7 +10,11 @@ <h2 class="govuk-label">You still need to provide details of your client’s fin
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <form class="form" action="/tasklist" method="post">
+      {% if mvp == true %}
```

Comment:
Is it just MVP? For non-MVP journey, when the user is passported on means and IOJ, they would also go to confirm details page rather than tasklist? If so, might be better to base this condition on passported for means?

Reply:
Yes @oliverweighell I - basing the condition on passported is better. If they are passported on means (regardless if MVP) they don't need to see the tasklist

Reply:
Done this in newest commit